Rd 308 utilities plugin docs wrapup (#1579)
cloudify-cosmo/docs.getcloudify.org: content/working_with/official_plugins/Configuration/utilities/files.md

        mode: 644
```

**Note:**

> If used in order to transfer file to remote host that defined in the blueprint use `cloudify.relationships.contained_in` relationship (see [openstack-blueprint.yaml](https://github.com/cloudify-community/blueprint-examples/blob/master/utilities-examples/cloudify_files/openstack-blueprint.yaml) example). 

Reviewer: ```suggestion
For file transfer to a remote host defined in the blueprint, use `cloudify.relationships.contained_in` relationship (see [openstack-blueprint.yaml](https://github.com/cloudify-community/blueprint-examples/blob/master/utilities-examples/cloudify_files/openstack-blueprint.yaml) example). 
```
